homeassistant/components/vera.py

```diff
@@ -164,7 +164,7 @@ def device_state_attributes(self):
         attr = {}
 
         if self.vera_device.has_battery:
-            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level + '%'
+            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level
```

Comment:
I don't think that this has the desired effect. Python does not allow concatenation of a number and a string, so the battery level itself must be a number in string form. So the attribute is still a string?